### PR TITLE
feat(STONEINTG-696): Issue fixes to onborad integration-service to RHTAP

### PR DESCRIPTION
**Pull request by hongweiliu17**
* let gitlint ignore the prs from rhtap bot
* rename AppStudioTestSucceededConditionPassed to AppStudioTestSucceededConditionSatisfied to pass snyk test

The error caught by snyk to be fix is as below 
```
        {
          "ruleId": "go/HardcodedPassword",
          "ruleIndex": 0,
          "level": "warning",
          "message": {
            "text": "Do not hardcode passwords in code. Found hardcoded saved in AppStudioTestSucceededConditionPassed.",
            "markdown": "Do not hardcode passwords in code. Found {0} saved in {1}.",
            "arguments": [
              "[hardcoded](0)",
              "[AppStudioTestSucceededConditionPassed](1)"
            ]
          },
          "locations": [
            {
              "physicalLocation": {
                "artifactLocation": {
                  "uri": "gitops/snapshot.go",
                  "uriBaseId": "%SRCROOT%"
                },
                "region": {
                  "startLine": 197,
                  "endLine": 197,
                  "startColumn": 12,
                  "endColumn": 49
                }
              }
            }
...
Task sast-snyk-check failed because of the following issues:
[
  "Do not hardcode passwords in code. Found hardcoded saved in AppStudioTestSucceededConditionPassed."
]
{"result":"FAILURE","timestamp":"1701929097","note":"For details, check Tekton task log.","namespace":"default","successes":0,"failures":2,"warnings":0}
```

## Maintainers will complete the following section

- [x] Commit messages are descriptive enough ([hints](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/))
- [x] Code coverage from testing does not decrease and new code is covered ([check the PR coverage on codecov](https://app.codecov.io/gh/redhat-appstudio/integration-service/pulls))
- [ ] [Controllers diagrams](https://github.com/redhat-appstudio/integration-service/tree/main/docs) are updated when PR changes controllers code  (if applicable)

Signed-off-by: Hongwei Liu <hongliu@redhat.com>

### CODE REVIEW

**Comment by MartinBasti** [comment]
Can we mention the error in commit message we are silencing here and why pls?

**Comment by hongweiliu17** [comment]
e2e reports  
`failed to unregister SprayProxy: error when unregistering PaC server https://pipelines-as-code-controller-openshift-pipelines.apps.rhtap-ocp-4-12-amd64-dthqs.ci.stonesoupengineering.com from SprayProxy server ...: failed to access SprayProxy server with status code: 503 and body: <html>` in log blob:https://prow.ci.openshift.org/f6b18a7e-d8d1-46c2-be36-120f01913f47

File: gitops/snapshot.go [hunk 1]
```diff
@@ -137,8 +137,8 @@ const (
 	// the global candidate list.
 	SnapshotAddedToGlobalCandidateListCondition = "AddedToGlobalCandidateList"
 
-	// AppStudioTestSucceededConditionPassed is the reason that's set when the AppStudio tests succeed.
-	AppStudioTestSucceededConditionPassed = "Passed"
+	// AppStudioTestSucceededConditionSatisfied is the reason that's set when the AppStudio tests succeed.
+	AppStudioTestSucceededConditionSatisfied = "Passed"
```
Comment (14rcole) [inline gitops/snapshot.go:141]: I don't think this name change really makes sense.  Is there any way to make  Snyk ignore all instances of `AppStudioTestSucceededConditionPassed`?  I know some tools have a line annotation option to manually override certain errors

Comment (dirgim) [reply gitops/snapshot.go:141]: Afaik, there's no way to ignore a specific instance of rule violation in snyk, only ignoring the entire file or ignoring the entire rule, none of which would be good practice imo.

That leaves renaming the constant as the best option, and though I agree it's tough to have a satisfactory replacement for `Passed`, the current one seems like a good compromise to me.

**Review by Josh-Everett — APPROVED** [review]

**Pull request merged**